sql/core/src/test/scala/org/apache/spark/sql/TPCDSQuerySuite.scala

```diff
@@ -339,6 +340,30 @@ class TPCDSQuerySuite extends BenchmarkQueryTest {
     }
   }
 
+  val tpcdsQueriesV2_7_0 = Seq(
+    "q1", "q2", "q3", "q4", "q5", "q5a", "q6", "q7", "q8", "q9", "q10", "q10a", "q11",
+    "q12", "q13", "q14_1", "q14_2", "q14a_1", "q14a_2",  "q15", "q16", "q17", "q18", "q18a", "q19",
+    "q20", "q21", "q22", "q22a", "q23_1", "q23_2", "q24_1", "q24_2", "q25", "q26", "q27", "q27a",
+    "q28", "q29", "q30", "q31", "q32", "q33", "q34", "q35", "q35a", "q36", "q36a", "q37", "q38",
+    "q39_1", "q39_2", "q40", "q41", "q42", "q43", "q44", "q45", "q46", "q47", "q48", "q49",
+    "q50", "q51", "q51a", "q52", "q53", "q54", "q55", "q56", "q57", "q58", "q59",
+    "q60", "q61", "q62", "q63", "q64", "q65", "q66", "q67", "q67a", "q68", "q69",
+    "q70", "q70a", "q71", "q72", "q73", "q74", "q75", "q76", "q77", "q77a", "q78", "q79",
+    "q80", "q80a", "q81", "q82", "q83", "q84", "q85", "q86", "q86a", "q87", "q88", "q89",
+    "q90", "q91", "q92", "q93", "q94", "q95", "q96", "q97", "q98", "q99")
+
+  tpcdsQueriesV2_7_0.foreach { name =>
+    val queryString = resourceToString(s"tpcds-v2.7.0/$name.sql",
```

Comment:
Could we just add the queries that are different from the v1.4 version?

Reply:
ok

Reply:
@maropu . It's great to have v2.7.
Could you check the schema too?
For example, we had better update [the following in the schema](https://github.com/apache/spark/pull/20343/files#diff-38fa80d1dc9860f07e135dd02d259269R247)?
```
-        |`web_country` STRING, `web_gmt_offset` STRING, `web_tax_percentage` DECIMAL(5,2))
+        |`web_country` STRING, `web_gmt_offset` DECIMAL(5,2), `web_tax_percentage` DECIMAL(5,2))
```

Reply:
ok, thanks.

Reply:
This is only related to test cases. Thus, it is fine Spark 2.3 release does not have it. You can do it in this PR.

Actually, this PR can be merged as long as we can fix all the issues.

Reply:
ok